src/main/java/salesforce/ui/pages/CasesFormPage.java

```diff
@@ -21,8 +21,42 @@
     private WebElement caseOriginMenu;
     @FindBy(xpath = "//span[text()='Priority']/../..//a[@class='select']")
     private WebElement priorityMenu;
+    @FindBy(xpath = "//span[text()='Type']/../..//a[@class='select']")
```

Comment:
Web elements with the same structure could be simplified to a single general String xpath and get the element from a method.